stretch_core/launch/stretch_realsense.launch.py

```diff
@@ -0,0 +1,75 @@
+import os
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration, PythonExpression
+from launch.conditions import IfCondition
+from launch_ros.actions import Node
+
+configurable_parameters = [{'name': 'resolution',                          'default': 'high', 'description': 'resolution of the color/depth images low or high'},                           
+                           {'name': 'publish_frustum_viz',                 'default': 'false', 'description': 'whether to pub viz of camera frustums'},
+                           {'name': 'publish_upright_img',                 'default': 'true', 'description': 'whether to pub rotated upright color image'},                           
+                          ]
+
+def declare_configurable_parameters(parameters):
+    return [DeclareLaunchArgument(param['name'], default_value=param['default'], description=param['description']) for param in parameters]
+
+def generate_launch_description():
+     publish_frustum_viz = LaunchConfiguration('publish_frustum_viz')
+     publish_upright_img = LaunchConfiguration('publish_upright_img')
+     resolution = LaunchConfiguration('resolution')
+
+     if resolution == 'high':
+          d435i_launch = IncludeLaunchDescription(
+               PythonLaunchDescriptionSource([os.path.join(
+                    get_package_share_directory('stretch_core'), 'launch'),
+                    '/d435i_high_resolution.launch.py'])
+               )
+     else:
+          d435i_launch = IncludeLaunchDescription(
+               PythonLaunchDescriptionSource([os.path.join(
+                    get_package_share_directory('stretch_core'), 'launch'),
+                    '/d435i_low_resolution.launch.py'])
+               )
```

Comment:
Replace the if/else with a f-string: `f'/d435i_{resolution}_resolution.launch.py'`